cmd/load-runner/runner.go

```diff
@@ -27,30 +27,35 @@ type portForwarder interface {
 	Stop() error
 }
 
+type metricsClient interface {
```

Comment:
[optional] `metricsFetcher`. a bit less generic, at least it suggests you can't add or remove metrics with it.